src/klein/test/test_resource.py

```diff
@@ -1153,3 +1155,12 @@ def test_afUnixSocket(self):
         self.assertIsInstance(server_port, int)
         self.assertIsInstance(path_info, unicode)
         self.assertIsInstance(script_name, unicode)
+
+
+if _PY3:
+    from sys import version_info as pyVersion
+
+    if pyVersion.major > 3 or (pyVersion.major == 3 and pyVersion.minor >= 5):
+        from os.path import join, dirname
+        name = join(dirname(__file__), "py3_test_resource.py")
```

Comment:
The name of the file implies all Py3 tests should go in there. This wouldn't be ideal considering Klein/Twisted is a single code base for Py2 and Py3. Maybe rename the file ``coroutine_tests.py`` or something along those lines.